clients/venice-client/src/test/java/com/linkedin/venice/fastclient/DispatchingAvroGenericStoreClientTest.java

```diff
@@ -174,7 +188,9 @@ private void setUpClient(
         batchGetValueFuture0.complete(batchGetResponse0);
         doReturn(batchGetValueFuture1).when(mockedTransportClient)
             .post(eq(REPLICA2_NAME + "/storage/test_store_v1"), any(), any());
-        batchGetValueFuture1.completeExceptionally(new VeniceClientException("Exception for client to return 503"));
+        if (!transportClientPartialIncomplete) {
```

Comment:
Does this mean if `transportClientPartialIncomplete` is set to true then future1 will never complete and we should still be able to read the result of future0?